task0/LinearRegression_Ferhat.ipynb

```diff
@@ -0,0 +1,1853 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    reg_2.score(X_test,Y_test)
</pre><p>You may print results in a more human readable format:</p><p><code>print(f"The regression score is ~ {reg_2.score(X_test, Y_test):.4f}")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>